tests/test_metrics/test_multi_label.py

```diff
@@ -149,6 +168,30 @@ def test_metric_interface_torch(metric_kwargs, preds, labels):
     assert isinstance(results, dict)
 
 
+@pytest.mark.parametrize('metric_kwargs', [{'num_classes': 3}])
+@pytest.mark.parametrize('preds', [
+    flow.Tensor([[0.1, 0.9, 0.8], [0.5, 0.6, 0.8]]),  # scores in a Tensor
+    # scores in Sequence
+    [flow.Tensor([0.1, 0.9, 0.8]), flow.Tensor([0.5, 0.6, 0.8])],
+    flow.Tensor([[0, 1, 0], [1, 1, 0]]),  # one-hot encodings in a Tensor
+    # one-hot encodings in Sequence
+    [flow.Tensor([0, 1, 0]), flow.Tensor([1, 1, 0])],
+    [flow.Tensor([1]), flow.Tensor([0, 1])],  # label-format in Sequence
+])
+@pytest.mark.parametrize('labels', [
+    flow.Tensor([[1, 0, 0], [0, 1, 0]]),  # one-hot encodings in a Tensor
+    # one-hot encodings in Sequence
+    [flow.Tensor([1, 0, 0]), flow.Tensor([0, 1, 0])],
+    [flow.Tensor([0]), flow.Tensor([1])],  # label-format in Sequence
+])
+@pytest.mark.skipif(flow is None, reason='OneFlow is not available!')
+def test_metric_interface_oneflow(metric_kwargs, preds, labels):
+    """Test all kinds of inputs in OneFlow."""
+    multi_label_metric = MultiLabelMetric(**metric_kwargs)
+    results = multi_label_metric(preds, labels)
+    assert isinstance(results, dict)
```

Comment:
```suggestion
@pytest.mark.parametrize('metric_kwargs', [{'num_classes': 3}])
@pytest.mark.parametrize('preds', [
    [[0.1, 0.9, 0.8], [0.5, 0.6, 0.8]],  # scores in a Tensor
    # scores in Sequence
    [[0.1, 0.9, 0.8], [0.5, 0.6, 0.8]],
    [[0, 1, 0], [1, 1, 0]],  # one-hot encodings in a Tensor
    # one-hot encodings in Sequence
    [[0, 1, 0], [1, 1, 0]],
    # [[1], [0, 1]],  # label-format in Sequence
])
@pytest.mark.parametrize('labels', [
    [[1, 0, 0], [0, 1, 0]],  # one-hot encodings in a Tensor
    # one-hot encodings in Sequence
    [[1, 0, 0], [0, 1, 0]],
    [[0], [1]],  # label-format in Sequence
])
@pytest.mark.skipif(flow is None, reason='OneFlow is not available!')
def test_metric_interface_oneflow(metric_kwargs, preds, labels):
    """Test all kinds of inputs in OneFlow."""
    multi_label_metric = MultiLabelMetric(**metric_kwargs)
    results = multi_label_metric(flow.Tensor(preds), flow.Tensor(labels))
    assert isinstance(results, dict)
```

Reply:
This is borrowed from [here](https://github.com/open-mmlab/mmeval/blob/main/tests/test_metrics/test_multi_label.py#L145), i think it would be better to keep the same style.

Reply:
Those unit tests will be refactored because they will fail if torch is not installed.